datalake/src/main/java/com/sequenceiq/datalake/service/sdx/DistroxService.java

```diff
@@ -65,7 +65,14 @@ public void restartAttachedDistroxClusters(String envCrn) {
         // We allow the refresh of DHs in NODE_FAILURE states to "fix" clusters that end up in this state due to their
         // services being unable to communicate with the DL due to a migration process or some other flow.
         Collection<StackViewV4Response> availableClusters = attachedDistroXClusters.stream()
-                .filter(cluster -> cluster.getStatus() == Status.AVAILABLE || cluster.getStatus() == Status.NODE_FAILURE)
+                .filter(cluster ->  {
+                    if (cluster.getStatus() == Status.AVAILABLE || cluster.getStatus() == Status.NODE_FAILURE) {
+                        return true;
+                    } else {
+                        LOGGER.info("Skipping restart for DH cluster: {}, Status: {}", cluster.getName(), cluster.getStatus().name());
```

Comment:
if we want to log all related DHs which cannot be restarted, I would collect them and log them into one log line using DEBUG level

Reply:
We need to log the cluster name and their status. 
I prefer to have them logged separately as we can not log all of them cleanly in one line.

If it were just the names, one logline would be fine.